pages/common/cal.md

```diff
@@ -2,18 +2,18 @@
 
 > Prints calendar information.
 
-- Display a calendar for the current month or specified month:
+- Display a calendar for the current month:
 
 `cal`
-`cal -m {{12}}`
-`cal -m {{Dec}}`
 
-- Display a calendar for the current year or a specified year:
+- Display a calendar for a specific month:
+
+`cal -m {{month_number}}`
+
+- Display all 12 months of calendar for a specific year:
 
-`cal -y`
 `cal 2013`
```

Comment:
While we're at it, if you don't mind, please change 2013 to {{year}}. And perhaps include a `cal -y` example before this one.